<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Pag_Usuario</class>
 <widget class="QMainWindow" name="Pag_Usuario">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>922</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btn_ord_num_letra">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>160</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Letra</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_ord_likes">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>160</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Likes</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_igual">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>130</y>
      <width>201</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Buscar igual</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_ord_fecha">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>160</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Fecha</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_empieza">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>70</y>
      <width>201</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Buscar Empieza</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_buscar_texto">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>40</y>
      <width>201</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Buscar Texto</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_buscar_texto">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>10</y>
      <width>281</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_temina">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>100</y>
      <width>201</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Buscar termina</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_no_contenido">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>160</y>
      <width>201</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Buscar no contenido</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_agregar">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>90</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Agregar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_back">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>220</y>
      <width>611</width>
      <height>384</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>609</width>
       <height>382</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="1">
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="1" column="0">
         <widget class="QPushButton" name="label_2">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="label_3">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QPushButton" name="label_1">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QPushButton" name="label_5">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QPushButton" name="label_6">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QPushButton" name="label_7">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="QPushButton" name="label_8">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="8" column="0">
         <widget class="QPushButton" name="label_9">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QPushButton" name="label_4">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="9" column="0">
         <widget class="QPushButton" name="label_10">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout_7">
        <item row="1" column="0">
         <widget class="QPushButton" name="label1__2">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QPushButton" name="label1__7">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QPushButton" name="label1__1">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QPushButton" name="label1__6">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="label1__3">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QPushButton" name="label1__5">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QPushButton" name="label1__4">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="9" column="0">
         <widget class="QPushButton" name="label1__10">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="8" column="0">
         <widget class="QPushButton" name="label1__9">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="QPushButton" name="label1__8">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>922</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
